### PR TITLE
[#116351243] Create myApp page and suyabay.readthedoc page

### PR DESCRIPTION
**Pull request by ademola-raimi**
#### What does this PR do?

Create myApp page and Suyabay documentation 
#### Description of Task to be completed?

When users are not logged in, it displays an appealing error page where users will have to either login or sign up in order to view _myApp_ page. Otherwise, if users are already logged in, it displays the page. 

On the _myApp_ page, users that are yet to create an app can click on the "create app" link where they can register and users that have already created an app will see a list of created apps. Though, this functionality has not been created yet.

Along with this, I create index.rst file which will contain the API documentation. This is quite different from the suyabay documentation which is the README.md.
#### How should this be manually tested?

Navigate to the suyabay/developer page, from this page, click on _Doc_ menu, this will navigate the user to the suyabay.readthedoc.com where the API documentation can be viewed.

Similarly, from suyabay/developer page, click on _MyApp_ menu, this will navigate the user to the suyabay/developer/myapp page. Where it either displays the appealing error message or the main page.
#### Any background context you want to provide?

None
#### What are the relevant pivotal tracker stories?
- #116351243
- #116173191
#### Screenshots (if appropriate)

screen shot 2016-04-01 at 11 46 16 am

screen shot 2016-04-01 at 11 46 33 am
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)

### CODE REVIEW

**Comment by unicodeveloper** [comment]
Please re-read the content of your answers to your PR questions and correct the grammatical blunders and errors @andela-araimi

File: index.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
+API Endpoints
+=============
+
+The Foursquare API provides methods for accessing a resource such as a
```
Comment (unicodeveloper) [inline index.rst]: When did Suyabay API become the Foursquare API?

File: index.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+--------------------
+
+As we announced in our August Update, the endpoints below will be
+deprecated by the end of 2014.
+
+| General Aspects Actions users leaderboard todos
+| venuegroups campaigns tips search
+| specials configuration retire campaigns list add timeseries end delete
+  start
+
```
Comment (unicodeveloper) [inline index.rst:38]: We don't have deprecated endpoints. Please remove this section.

File: index.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+API Endpoints
+-------------
+
+An overview of the entire API and its endpoints. Read these API docs to
+find out more about leaving tips, checking in, seeing what your friends
+are up to, and of course, searching for and viewing venue info.
+
+Real-time API
+-------------
```
Comment (unicodeveloper) [inline index.rst:48]: We also don't have a real-time API

File: index.rst [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+--------------
+
+Just need a database of places? The Venues Service allows developers to
+search for places and access a wealth of information about them,
+including addresses, popularity, tips, and photos. It’s available free
+and without any user authentication, as long as applications include
+adequate attribution.
+
+Merchant Platform
+-----------------
```
Comment (unicodeveloper) [inline index.rst]: Please eliminate multiple copies of the same thing in this file. You duplicated the Foursquare readme and you still have multiple copies of the same section

File: public/css/api.main.css [hunk 5]
```diff
@@ -219,17 +243,61 @@ body, html{
     text-align: center;
 }
 
-.waves-light {
+/*.waves-effect {
+    vertical-align: top!important;
+}*/
```
Comment (unicodeveloper) [inline public/css/api.main.css]: **Nitpick**

Please remove unused code totally.

File: resources/views/api/includes/contents/app.blade.php [hunk 6]
```diff
@@ -1,22 +1,49 @@
-<!-- top nav -->
-<div class="row">
+@if ( Auth::check() )
+
+        <!-- top nav -->
     @include("api.includes.sections.top_nav_user")
```
Comment (unicodeveloper) [inline resources/views/api/includes/contents/app.blade.php]: **Code Quality - Nitpick**

Repetition. This is included in the if and else. Shouldn't the view be included outside the if & else block?

File: resources/views/api/includes/sections/top_nav_user.blade.php [hunk 7]
```diff
@@ -9,20 +10,6 @@
...
-                                <i class="material-icons teal-text text-lighten-2">search</i>
-                            </label>
-                            <button type="submit" style="display:none;" name="search">search</button>
-                            <i class="material-icons">close</i>
-                        </div>
-                    </form>
-                </li>
-
-                <li>
                     @if (  Auth::check() )
```
Comment (unicodeveloper) [inline resources/views/api/includes/sections/top_nav_user.blade.php]: **Nitpick**

Please remove the extra spaces around Auth::check()..It should rather be `@if (Auth::check())`

File: resources/views/api/includes/sections/top_nav_user.blade.php [hunk 8]
```diff
@@ -1,3 +1,4 @@
+<div class="row">
 <div class="navbar-fixed">
     <nav role="navigation">
         <div class="nav-wrapper">
```
Comment (unicodeveloper) [inline resources/views/api/includes/sections/top_nav_user.blade.php]: Where is this particular file being used in this project ?.. `top_nav_user.blade.php`

**Comment by ademola-raimi** [comment]
I haven't compiled the documentation yet, this was created in order to view how the readthedoc works

**Comment by unicodeveloper** [comment]
Compiled or completed ? @andela-araimi

**Comment by ademola-raimi** [comment]
I mean Compiled, I still need information from @andela-tolotin. I will probably sync with @andela-tolotin so that we have a complete and clear documentation

**Comment by unicodeveloper** [comment]
Ohh okay, but please remove the duplicates and still make it presentable, there is a cool acceptable format of documentation, let it not be too scattered @andela-araimi

**Comment by ademola-raimi** [comment]
acknowledged

**Comment by andela-oadebayo** [comment]
**Code-Quality- Critical**
@andela-araimi work on making the api.min.css file Dry (especially the repeating Margin properties)

**Comment by andela-oadebayo** [comment]
**Code-Quality - Critical**
@andela-araimi Combine the two nav files that you have for the API into one, to make the code more Dry.

File: public/css/api.main.css [hunk 9]
```diff
@@ -4,10 +4,18 @@ body, html{
...
 
+#settings {
+    -webkit-box-shadow: -1px 2px 16px 3px rgba(0,0,0,0.52);
+    -moz-box-shadow: -1px 2px 16px 3px rgba(0,0,0,0.52);
+    box-shadow: -1px 2px 16px 3px rgba(0,0,0,0.52);
+    margin-top: 65px;
+    height: 100px;
+}
+
 #hero{
```
Comment (andela-oadebayo) [inline public/css/api.main.css]: **Code Quality - NitPick**
@andela-araimi there should be space between the id name and the opening curly braces. Fix any other appearance of this in this file.

File: public/css/api.main.css [hunk 10]
```diff
@@ -164,3 +164,140 @@ body, html{
...
+===============================================
+*/
+
+#main-one {
+    clear: both;
+    min-height: 400px;
+    width: 80%;
+    margin-bottom:20px;
+    background-color: #fff;
+    margin-right: auto;
```
Comment (andela-oadebayo) [inline public/css/api.main.css]: **Code Quality - NitPick**
@andela-araimi Make this CSS file more dry, by making use of the margin only declaration.

**Comment by andela-oadebayo** [comment]
**Code Quality - Nitpick**
@andela-araimi As discussed earlier let's make use of only one nav file for the API

File: resources/views/api/includes/contents/newapp.blade.php [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+    <div class='row'>
+    	<div class="col s8">
+        	<div id="main-new-one">
+	    		<div class='form-feature'>
+			    	<form>
+
+			    		<h5>Application name</h5>
+
+				    		<div class='form-group'>
```
Comment (andela-oadebayo) [inline resources/views/api/includes/contents/newapp.blade.php]: **Code Quality - Nitpick**
@andela-araimi Checkout Laravel Form helpers (Input::old), for better user experinece.

**Comment by unicodeveloper** [comment]
What's the latest on this @andela-araimi and what's delaying this PR @andela-oadebayo ?

**Pull request merged**